### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by phaenyxfeathers** [rgbkrk/atom-script#266]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app/src/atom.js:55:11)
ScriptView.initialize (/Users/innafinit/.atom/packages/script/lib/script-view.coffee:28:9)
```

**Comment by erran** [comment]
@phaenyxfeathers why'd you decide to close the issue? I see the deprecation warnings on my box.

**Comment by erran** [comment]
Fixed by #407, new release pending.
